x-pack/plugins/ml/public/application/explorer/add_to_dashboard_control.tsx

```diff
@@ -0,0 +1,339 @@
+    return {
+      onChange: ({ queryText }) => {
+        setIsLoading(true);
+        fetchDashboards(queryText);
+      },
+      box: {
+        incremental: true,
+      },
+    };
+  }, []);
+
+  const addSwimlaneToDashboardCallback = useCallback(
+    async (item: DashboardItem) => {
+      setAddedDashboards({ ...addedDashboards, [item.id]: 'pending' });
+
+      const panelsData = Object.entries(selectedSwimlanes)
+        .filter(([, isSelected]) => isSelected)
+        .map(([swimlaneType]) => {
+          const config = getDefaultEmbeddablepaPanelConfig(jobIds);
+          if (swimlaneType === SWIMLANE_TYPE.VIEW_BY) {
+            return {
+              ...config,
+              embeddableConfig: {
+                jobIds,
+                swimlaneType,
+                viewBy,
+                limit,
+              },
+            };
+          }
+          return {
+            ...config,
+            embeddableConfig: {
+              jobIds,
+              swimlaneType,
+            },
+          };
+        });
+
+      try {
+        await dashboardService.attachPanels(item.id, item.attributes, panelsData);
+        setAddedDashboards({ ...addedDashboards, [item.id]: 'success' });
+      } catch (e) {
+        toasts.danger({
+          body: e,
+        });
+      }
+    },
+    [addedDashboards, selectedSwimlanes]
+  );
+
+  const columns: EuiTableProps['columns'] = useMemo(
+    () => [
+      {
+        field: 'title',
+        name: i18n.translate('xpack.ml.explorer.dashboardsTable.titleColumnHeader', {
+          defaultMessage: 'Title',
+        }),
+        sortable: true,
+        truncateText: true,
+      },
+      {
+        field: 'description',
+        name: i18n.translate('xpack.ml.explorer.dashboardsTable.descriptionColumnHeader', {
+          defaultMessage: 'Description',
+        }),
+        truncateText: true,
+      },
+      {
+        field: 'actions',
+        name: i18n.translate('xpack.ml.explorer.dashboardsTable.actionsColumnHeader', {
+          defaultMessage: 'Actions',
+        }),
+        align: 'right',
+        actions: [
+          {
+            name: i18n.translate('xpack.ml.explorer.dashboardsTable.quickAddLabel', {
+              defaultMessage: 'Quick add',
+            }),
+            render: (item) => {
+              return addedDashboards[item.id] === 'pending' ? (
+                <EuiLoadingSpinner size="m" />
+              ) : (
+                <EuiToolTip
+                  content={
+                    addedDashboards[item.id] === 'success' ? (
+                      <FormattedMessage
+                        id="xpack.ml.explorer.dashboardsTitle.successfullyAddedMessage"
+                        defaultMessage="Successfully added"
+                      />
+                    ) : (
+                      <FormattedMessage
+                        id="xpack.ml.explorer.dashboardsTable.quickAddLabel"
+                        defaultMessage="Quick add"
```

Comment:
Maybe just 'Add to dashboard' ?

Reply:
Changed in https://github.com/elastic/kibana/pull/68784/commits/eef0cc78c532ebd58b36ff1e12fda3c029ca25b6``